.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+#### Description
+
+Add a brief description of your changes, so every one new to the project
+understands it too.
+
+#### Purpose
+
+Explain the purpose of the Pull Request, if possible. Why you added this
+feature, or why you have done the changes in the way you did?
+
+#### TODO
+
+If you can't met the requirements of the table below, create a short todo
+list and add the relevant parts:
+
+- [ ] fix the tests as they have not been updated yet
+- [ ] submit changes to the documentation
+- [ ] document the BC breaks
+
+If the code is not finished yet because you don't have time to finish it or
+because you want early feedback on your work, add an item to the todo list
+and add `[WiP]` to the title of your Pull Request:
+
+- [ ] finish the code
+- [ ] gather feedback for my changes
+
+Don't add this todo part if your Pull Request is finished and has no todos!
+
+#### Summary
+
+| Q | A
+| --- | ---
+| Bug fix? | [yes|no]
+| New feature? | [yes|no]
+| BC breaks? | [yes|no]
+| Deprecations? | [yes|no]
+| Tests pass? | [yes|no]
+| Fixed tickets | [comma separated list of tickets fixed by the PR -> `fixes #num`]
+| License | MIT
```

Comment:
The license is required to make sure all contributions are licensed under MIT. Otherwise it is a little bit unsure, which license a contribution has.

Another solution would be to require signing contribution license agreements, but IMHO the license line in the PR should be preferred.